### ISSUE TITLE
Unchecking registerUserOnOrderComplete doesn't work

### ISSUE THREAD

**Issue by gijsstegehuis** [craftcms/commerce#1299]
### Description
CartController's `actionUpdateCart()` doesn't allow unchecking `registerUserOnOrderComplete` option.

see this line:
https://github.com/craftcms/commerce/blob/3.1/src/controllers/CartController.php#L200

Is there some other way to achieve this?

### Additional info

- Craft version: 3.4.6.1
- PHP version: 7.3.14
- Database driver & version: -
- Plugins & versions: -

**Comment by lukeholder** [comment]
Thanks for reporting. Will fix.

**Comment by lukeholder** [comment]
This has been fixed in 353d4a4da8c7f95591700d0dd706d11dad45da5d and will be included in the `3.1` release.

To unset the `registerUserOnOrderComplete` you can pass the value of `false` in a hidden input.

```twig
<label for="registerUserOnOrderComplete">
  <input type="hidden" name="registerUserOnOrderComplete" value="false"/>
  <input type="checkbox" id="registerUserOnOrderComplete" name="registerUserOnOrderComplete" value="1" /> {{ "Register me for a user account"|t }}
</label>
```
